### DISCUSSION TITLE
Yarp bindings  python

### DISCUSSION THREAD

**Discussion by robertorovella91** [robotology/community#283] [Q&A] [answered]
Hi,

We are trying to use Python in order to control the iCub, but after installing with python3.5 the libraries, following the instructions in the README, the example files in `yarp/bindings/python/examples` don't start and this error occurs:
```
AttributeError: module 'yarp' has no attribute 'Network'
```

How can we solve this problem? 

Thanks in advance,
Roberto

**Comment by traversaro** [comment]
Hi @robertorovella91, 
* what is the content of your  [`PYTHONPATH`](https://docs.python.org/3/using/cmdline.html#envvar-PYTHONPATH) environmental variable? 
* Which OS are you using?

**Comment by robertorovella91** [comment, marked as answer]
Hi @traversaro 
The problem was that we had not setted PYTHONPATH with the folder containing yarp.so file. Now we have fixed and solved the problem.
Thank you.

**Comment by zmd302** [comment]
I am in same problem, (module 'yarp' has no attribute 'Network')
I added to path, but not solved,
![image](https://user-images.githubusercontent.com/38297825/65777088-c59aad00-e15c-11e9-9001-1608ed76adf0.png)

**Comment by zmd302** [comment]
expecting soon response,.. windows 10

**Comment by alecive** [comment]
Me too, thanks, I will add another message in 10 minutes in case you haven't seen this.

**Comment by alecive** [comment]
Ops sorry I meant 10 seconds.. here's my second message.

**Comment by zmd302** [comment]
@robertorovella91
Hi I no folder containing yarp.so file
